expo-zustand-styled-components/src/components/IssuePullRequestCard/IssuePullRequestCard.tsx

```diff
@@ -10,32 +13,35 @@ import {
   TitleLabelsWrapper,
   Labels,
 } from './IssuePullRequestCard.styles';
-import IssuesIcon from '../Icons/IssuesIcon';
+
 import { STATES, cardTypes } from './data';
+
 import ClosedIssueIcon from '../Icons/ClosedIssueIcon';
-import { colors } from '../../utils/style-variables';
 import PullRequestIcon from '../Icons/PullRequestIcon';
 import MergedPrIcon from '../Icons/MergedPrIcon';
 import ClosedPrIcon from '../Icons/ClosedPrIcon';
-import { Text, View } from 'react-native';
-import { format } from 'date-fns';
-import { Link } from '@react-navigation/native';
 import CommentIcon from '../Icons/CommentIcon';
+import IssuesIcon from '../Icons/IssuesIcon';
+
```

Comment:
The changes in this file have nothing to do with pagination, why make the changes?